pkg/handler/handler.go

```diff
@@ -116,11 +118,13 @@ func (o *OpenAPIService) UpdateSpec(openapiSpec *spec.Swagger) (err error) {
 		}
 		return []byte(computeETag(json)), nil
 	})
-	o.lastModified = time.Now()
-
 	return nil
 }
 
+func (o *OpenAPIService) UpdateSpec(openapiSpec *spec.Swagger) (err error) {
```

Comment:
I guess I am confused about how this actually defers anything if this function is still getting called?

Reply:
Well, UpdateSpec continues to work like it did, but there is a new `UpdateSpecLazy` that allows you to give a method rather than a resolved spec, which isn't used yet in k/k.

Reply:
Ah the call comes from k/k? got it